parser/grammar/nonreservedwords.txt

```diff
@@ -0,0 +1,353 @@
+void non_reserved_word():
+{}
+{
+    "A"
+  | "ABSOLUTE"
```

Comment:
Does the ones that are reserved words in spec but we changed to non-reserved belong here too?